lib/lita/adapters/slack/slack_im.rb

```diff
@@ -4,8 +4,9 @@ class Slack < Adapter
       # @api private
       class SlackIM
         class << self
-          def from_data_array(ims_data)
-            ims_data.map { |im_data| from_data(im_data) }
+          def from_data_array(channels_data)
+            channels_data['channels'].select { |channel_data| channel_data['is_im'] }
```

Comment:
I'm not sure if we should also be checking for `channel_data['is_mpim']` here??

Reply:
iiuc mpim replaced what the older APIs called group messages.